applications/DEMApplication/custom_elements/cluster3D.cpp

```diff
@@ -60,6 +60,9 @@ namespace Kratos {
         RigidBodyElement3D::Initialize(r_process_info);
 
         const double cl = GetGeometry()[0].FastGetSolutionStepValue(CHARACTERISTIC_LENGTH);
+	if(!GetProperties().Has(CLUSTER_INFORMATION)) {
```

Comment:
```suggestion
	KRATOS_ERROR_IF_NOT(GetProperties().Has(CLUSTER_INFORMATION)) {
```

Reply:
Thanks @philbucher , I will change this, but removing the curly brackets...